### ISSUE TITLE
Tests are not included in published crate

### ISSUE THREAD

**Issue by jamessan** [rust-lang/cc-rs#1192]
Since https://github.com/rust-lang/cc-rs/pull/934, the tests are no longer included in the crate. However, some distributors (like Debian) create their packages from the crate rather than the git repo. This means Debian can no longer easily run the crate's tests to validate updates of the package.

Could the tests please be restored to the crate?

**Comment by NobodyXu** [comment]
Yes, however note that many tests sit in an unpublished workspace https://github.com/rust-lang/cc-rs/tree/main/dev-tools/cc-test

so if you'd like to test it, we would also have to publish cc-test as well
